scripts/utils/migration_scripts/collectionsMigration/main.go

```diff
@@ -0,0 +1,219 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type config struct {
+	teamsDir,
+	collectionDir,
+	collectionCopyDir string
+	migration bool
+}
+
+type teamCollection struct {
+	id   string
+	name string
+}
+
+type collection struct {
```

Comment:
How can we be sure this is the complete list of fields? Any fields not defined in this list will be lost on Marshalling into the struct and then won't be left in the resulting json file. I suggested we should use jq for modifying the files originally as it would preserve all other fields and only modify the one field we wanted to change. I'd still suggest doing this now even if we use something like `interface{}` within go and then iterate to find the field rather than Marshalling the struct explicitly like this.